### PR TITLE
Issue: 191 - Add more icons for cuisines and attire

### LINKED ISSUE #191: Product Pages: Fix Attire SVGs
E.g., https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/structured-content/restaurants/cafe-americano
Screen Shot 2023-04-20 at 3 32 58 PM

Casual should have a t-shirt icon as seen in: https://www.caesars.com/caesars-palace/restaurants/cafe-americano

Similar issue seen for Formal & Business Casual attires.

Also, cuisine : `American` should be a burger instead of a fork and a knife.

### PR DESCRIPTION
**Pull request by swetabar**
Fix #191 

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/structured-content/restaurants/cafe-americano
- After: https://issue-191-fix-attire-svgs--caesars--hlxsites.hlx.page/caesars-palace/drafts/structured-content/restaurants/cafe-americano

## 📝 Description:
  
Screen Shot 2023-04-21 at 8 11 31 AM

### CODE REVIEW

File: caesars-palace/icons/business-casual.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="injected-svg" data-src="/content/dam/empire/icon-library/fontawesome-icons/user-tie.svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path fill="#fff" d="M224 288c79.5 0 144-64.5 144-144S303.5 0 224 0 80 64.5 80 144s64.5 144 144 144zm0-240c52.9 0 96 43.1 96 96s-43.1 96-96 96-96-43.1-96-96 43.1-96 96-96zm91.9 256.2l-56.5 154.5L240 376l32-56h-96l32 56-19.5 82.7L132 304.2C58.9 305.5 0 365 0 438.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-25.6c0-73.4-58.9-132.9-132.1-134.2zM96 464H48v-25.6c0-35.4 21.9-66.2 53-79.4l38.4 105H96zm304 0h-91.3L347 359c31 13.2 53 44 53 79.4V464z"></path></svg>
```
Comment (tmathern) [inline caesars-palace/icons/business-casual.svg:1]: We already have this one, it's https://github.com/hlxsites/caesars/blob/main/caesars-palace/icons/dress-code-smart-business-casual.svg. We should just rename it.

File: caesars-palace/icons/casual.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" class="injected-svg" data-src="/content/dam/empire/icon-library/fontawesome-icons/tshirt.svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path fill="#fff" d="M638 121c-3.3-9.8-10.2-17.8-19.5-22.4L420.2 0c-9.5 13.2-28.4 50.3-100.2 50.3-72.4 0-91.1-37.7-100.2-50.3L21.6 98.6C12.3 103.2 5.3 111.2 2 121c-3.3 9.9-2.6 20.4 2.1 29.7l53 106.2c9.6 19.2 33 27 51.6 17.7l24-11.3c5.3-2.5 11.4 1.4 11.4 7.2v185.3c0 31 25.1 56.2 56 56.2h240c30.9 0 56-25.2 56-56.2V270.6c0-5.9 6.1-9.7 11.4-7.2l23.5 11.1c19.1 9.7 42.5 1.8 52.1-17.4l53-106.2c4.4-9.5 5.2-20 1.9-29.9zm-94 106.4l-73.2-34.6c-10.6-5-22.8 2.7-22.8 14.5v248.6c0 4.4-3.6 8-8 8H200c-4.4 0-8-3.6-8-8V207.3c0-11.7-12.2-19.5-22.8-14.5L96 227.4l-44.8-89.9 155.5-77.3c26.4 24 67.8 38.3 113.3 38.3s86.9-14.3 113.2-38.2l155.5 77.3-44.7 89.8z"></path></svg>
```
Comment (tmathern) [inline caesars-palace/icons/casual.svg:1]: We already have it, should just rename https://github.com/hlxsites/caesars/blob/main/caesars-palace/icons/dress-code-casual.svg

File: caesars-palace/icons/american.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="injected-svg" data-src="/content/dam/empire/icon-library/fontawesome-icons/hamburger.svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path fill="#fff" d="M352 176a16 16 0 1 0-16-16 16 16 0 0 0 16 16zm-96-32a16 16 0 1 0-16-16 16 16 0 0 0 16 16zm-96 32a16 16 0 1 0-16-16 16 16 0 0 0 16 16zm352 112a79.45 79.45 0 0 0-28.07-60.4c.36-.54.85-.92 1.2-1.48a72.63 72.63 0 0 0 .63-75.43C442.33 78.69 352.18 32.09 256 32c-96.11.09-186.26 46.67-229.7 118.67a72.65 72.65 0 0 0 .6 75.44c.35.55.85.94 1.21 1.49a79.32 79.32 0 0 0 5.65 125.46c-.66 2.83-1.73 5.51-1.73 8.53v34.68A83.82 83.82 0 0 0 115.72 480h280.56A83.82 83.82 0 0 0 480 396.27v-34.68c0-3-1.07-5.7-1.73-8.53A79.75 79.75 0 0 0 512 288zM67.37 175.46C102.3 117.56 176.32 80.09 256 80c79.69.09 153.75 37.57 188.69 95.47a24.59 24.59 0 0 1-.21 25.17c-2.93 4.68-7.38 7.36-12.2 7.36H79.75c-4.82 0-9.26-2.69-12.2-7.37a24.56 24.56 0 0 1-.18-25.17zM432 396.27A35.77 35.77 0 0 1 396.28 432H115.72A35.77 35.77 0 0 1 80 396.27v-25.6h352zm0-76.27H80a32 32 0 0 1 0-64h352a32 32 0 0 1 0 64z"></path></svg>
```
Comment (tmathern) [inline caesars-palace/icons/american.svg:1]: We already have it - needs renaming: https://github.com/hlxsites/caesars/blob/main/caesars-palace/icons/restaurant-american.svg

**Review by tmathern — CHANGES_REQUESTED** [review]

File: caesars-palace/icons/american.svg (hunk 3 above)
Comment (swetabar) [reply caesars-palace/icons/american.svg]: Done!

File: caesars-palace/icons/casual.svg (hunk 2 above)
Comment (swetabar) [reply caesars-palace/icons/casual.svg]: Done!

File: caesars-palace/icons/business-casual.svg (hunk 1 above)
Comment (swetabar) [reply caesars-palace/icons/business-casual.svg]: Done!

**Review by amol-anand — APPROVED** [review]

**Review by amol-anand — APPROVED** [review]

**Pull request merged**